app/tests/util/webhooks.py

```diff
@@ -0,0 +1,55 @@
+import time
+import os
+from typing import List, Literal
+
+from httpx import AsyncClient
+import httpx
+
+from app.tests.util.string import get_wallet_id_from_JWT
+
+WEBHOOKS_URL = os.getenv("WEBHOOKS_URL", "http://localhost:3010")
+
+topics = Literal[
+    "connections",
+    "issue_credential",
+    "forward",
+    "ping",
+    "basicmessages",
+    "issuer_cred_rev",
+    "issue_credential_v2_0",
+    "issue_credential_v2_0_indy",
+    "issue_credential_v2_0_dif",
+    "present_proof",
+    "revocation_registry",
+]
+
+
+def check_webhook_state(
+    client: AsyncClient,
+    desired_state: dict = None,
+    topic: topics = "connections",
```

Comment:
Ah I see you also used the `connections` topic as default here. I think if we make the property required the api of the method would be more clear (I don't have to look in the implementation what the topic is if I don't provide one)